### ISSUE TITLE
Workshop for Accessibility Professionals Outside of W3C

### ISSUE THREAD

**Issue by lolaodelola** [lolaodelola/accessibility-compat-data#5]
This is for accessibility professionals that either want to be involved or should be involved but don't engage with the W3C working group structure. This would also include people in various W3C community groups. See Logseq (ACD: Non-W3C Invite List) for list of names.

## What Do I Want To Know?
Same as #2 

## Work Breakdown

- [ ] Compose list of invitees
- [ ] Schedule workshop
- [ ] Send out invites
- [ ] Plan workshop
- [ ] Write up session

## Deliverables
1. Workshop
2. Summary of session
